docs/docs/flags/enrichment.1.md

```diff
@@ -0,0 +1,127 @@
+---
+title: TRACEE-ENRICHMENT
+section: 1
+header: Tracee Enrichment Flag Manual
+date: 2025/12
+...
+
+## NAME
+
+tracee **\-\-enrichment** - Configure enrichment for container events and other enrichment options
+
+## SYNOPSIS
+
+tracee **\-\-enrichment** [container.enabled|container.cgroup.path=*path*|container.cgroup.force|container.docker.socket=*socket_path*|container.containerd.socket=*socket_path*|container.crio.socket=*socket_path*|container.podman.socket=*socket_path*|resolve-fd|exec-hash.enabled|exec-hash.mode=*mode*|user-stack-trace] [**\-\-enrichment** ...]
+
+## DESCRIPTION
+
+The `--enrichment` flag allows you to configure enrichment options for container events and other enrichment features.
+
+### Flags
+
+- **container.enabled**
+  Enable container enrichment. Presence of the flag enables it, absence disables it.
+  Example:
+  ```console
+  --enrichment container.enabled
+  ```
+
+- **container.cgroup.path**=*path*
```

Comment:
Now that I'm reading it, I think it will be more clear if we will call it `container.cgroupfs` instead of `container.cgroup`. WDYT?